doc/source/management/data_at_rest_encryption.rst

```diff
@@ -190,6 +190,32 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+Temporary file encryption
+=========================
+
+A new feature, implemented since |Percona Server| :rn:`5.7.22-22`, is
+encryption of temporary files, triggered by the :variable:`encrypt-tmp-files`
+option.
+
+For each temporary file, an encryption key is generated locally, only kept
```

Comment:
Please, also specify in which scenarios temporary files are used in Percona Server:
- filesort (say, SELECTs with SQL_BIG_RESULT hints)
- binary log transactional caches
- Group Replication caches